packages/dds/merge-tree/src/client.ts

```diff
@@ -20,7 +20,7 @@ import { UsageError } from "@fluidframework/container-utils";
 import { IIntegerRange } from "./base";
 import { RedBlackTree } from "./collections";
 import { UnassignedSequenceNumber, UniversalSequenceNumber } from "./constants";
-import { LocalReferencePosition } from "./localReference";
+import { LocalReferencePosition, SlidingPreference } from "./localReference";
```

Comment:
i would prefer if we split these PRs into a merge tree portion and an interval portion